server/tests/utils.py

```diff
@@ -51,4 +63,55 @@ def build_questions(uid, n=5):
     questions = [None] * n
     for i in range(n):
         questions[i] = build_question(f"Question {i+1}", uid)
-    return questions
\ No newline at end of file
+    return questions
+
+
+def generate_sample_webm(question_id):
+    """
+    Generate a sample webm file from a question id for testing 
+    """
+    command = [
+        "ffmpeg",
+        "-f",
+        "lavfi",
+        "-i",
+        "sine=frequency=1000:duration=5",
+        f"{TMP_DIR}/{question_id}{WEBM_EXT}"
+    ]
+    subprocess.run(command, stdout=subprocess.PIPE, stdin=subprocess.PIPE)
+
+    return retrieve_sample_webm(question_id)
+
+
+def retrieve_sample_webm(question_id):
+    """
+    Retrieve generated webm sample from question_id
+    """
+    file = f"{TMP_DIR}/{question_id}{WEBM_EXT}"
+
+    webm = FileStorage(
+        stream=open(file, "rb"),
+        filename=f"{question_id}{WEBM_EXT}",
+        content_type="video/webm"
+    )
+    
+    return webm
+
+
+def cleanup_webm(uid, question_id):
+    """
+    Remove local .webm file and delete .wav file from GCS bucket
+    """
+    local_path = f"{TMP_DIR}/{question_id}{WEBM_EXT}"
+    if os.path.exists(local_path):
+        os.remove(local_path)
+
+    gcs_path = f"{uid}/{question_id}{WAV_EXT}"
+    delete_file(GCS_BUCKET, gcs_path)
```

Comment:
This is a very good idea to keep the test data not accumulating over time. Can you add deletion of the local .wav file we get after converting webm to wav using ffmpeg. I believe it's the same name but just a different extension.

Moreover, there is a util function in `server/app/utils/misc.py` named `delete_local_file` which you could use to abide by the DRY principle